docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -170,6 +170,9 @@ would produce the following ConfigMap:
 
 ```shell
 kubectl get configmap game-config-env-file -o yaml
+```
+
+```yaml
```

Comment:
-1
This is output of shell. So it's part of shell.

Reply:
I'm following the pattern used in the first section, see [L73-L77](https://github.com/kubernetes/website/blame/master/docs/tasks/configure-pod-container/configure-pod-configmap.md#L73-L77).

It greatly improves readability by nicely separating input from output and doing the correct syntax highlighting for the output.

Reply:
OK, this makes sense to me.

Reply:
Yes, input and output should be separated. Also makes it easier when users are copying the shell command.